### PR TITLE
deb: move some logic away from packager/postinst

### LINKED ISSUE #2023: Move some packager logic in the debian stuff
We are using some hacks to build the deb packages, like moving the `openquake.cfg` and other stuff into `openquake/engine/`. This should be done by the `debian/*` stuff during the build phase (i.e. `/etc/openquake/` is missing in the deb and the cfg is taken from `lib/python` during the `postinst`).

This issue forced us to make ugly patches and currently breaks local builds made with `dpkg-buildpackage` without using the full `packager.sh` infrastructure (which is complex and sometime useless if immediate testing is not needed).

### PR DESCRIPTION
**Pull request by daniviga**
Removes some "manual" steps in the packager required to have the deb package and also removes some logic from the postinst.

Ubuntu: https://ci.openquake.org/job/zdevel_oq-engine/1814/
RedHat: https://ci.openquake.org/job/redhat_oq-engine/27/

Closes #2023

### CODE REVIEW

**Comment by micheles** [comment]
LGTM

File: debian/postinst [hunk 1]
```diff
@@ -52,30 +52,10 @@ if [ $(cat /etc/passwd | grep ^openquake: | wc -l) -eq 0 ]; then
...
 
-HDIR=/usr/share/openquake/engine
-IDIR=/usr/lib/python2.7/dist-packages/openquake/engine
-mkdir -p $HDIR
-
-rm -f $HDIR/celeryconfig.py.new_in_this_release
-cp $IDIR/celeryconfig.py $HDIR
+HDIR=/usr/share/openquake
+mkdir -p ${HDIR}/engine
 chown -R $USER.$USER $HDIR
```
Comment (nastasi-oq) [inline debian/postinst]: With HDIR=/usr/share/openquake you are changing things that potentially are managed by other packages, why this change ?

Comment (daniviga) [inline debian/postinst]: You are right.

Comment (daniviga) [inline debian/postinst]: Fixed.

**Comment by nastasi-oq** [comment]
LGTM
